src/doc/src/guide/continuous-integration.md

```diff
@@ -21,6 +21,41 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+### GitHub Actions
+
+To test your package on GitHub Actions, here is a sample `.github/workflows/ci.yml` file:
+
+```yaml
+name: Cargo Build & Test
+
+on:
+  push:
+
+jobs:
+  build_and_test:
+    name: Rust project - latest
+    runs-on: ${{ matrix.operating-system }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        toolchain:
+          - stable
+          - beta
+          - nightly
+        operating-system:
+          - "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.toolchain }}
```

Comment:
This does not change the default toolchain, so all of subsequent cargo commands will use the stable cargo installed in Github Actions runner.

Reply:
Thanks for spotting that. I  have updated it with the `override: true` flag to correct this. https://github.com/SamMorrowDrums/rust-action-test/runs/4559440996?check_suite_focus=true